core/src/main/resources/jenkins/security/ResourceDomainRecommendation/message_ru.properties

```diff
@@ -0,0 +1,6 @@
+blurb = Content-Security-Policy по умолчанию сейчас переопределяется системным свойством <code>hudson.model\
```

Comment:
Политика Content-Security-Policy по умолчанию в настоящее время переопределяется с помощью системного свойства <code>hudson.model.DirectoryBrowserSupport.CSP</code>, что является потенциальной проблемой безопасности при просмотре ненадежных файлов.

Reply:
I was trying to avoid very long winding wordy sentences and rather split them, they feel much more wordy in Russian and harder to read somehow. But I think I also can take a few words tomorow